modules/ding_availability/js/ding_availability_labels.js

```diff
@@ -138,7 +138,11 @@
           group = $('.js-unavailable', groups_wrapper);
 
           if (group.length === 0) {
-            group = $('<p class="js-unavailable">' + Drupal.t('Not available') + ': </p>');
+            var msg = Drupal.t('Not available');
+            if (Drupal.settings.ding_ill) {
```

Comment:
Nitpicking: if i understand it this "ding_ill" is a new module not part of core. So would the right check be:

`if (Drupal.settings.hasOwnProperty('ding_ill))`

Reply:
Good point, I will update it.